Document/0x06a-Platform-Overview.md

```diff
@@ -18,9 +18,9 @@ In the past, sideloading was possible only with a jailbreak or complicated worka
 
 iOS apps are isolated from each other via Apple's iOS sandbox (historically called Seatbelt), a mandatory access control (MAC) mechanism describing the resources an app can and can't access. Compared to Android's extensive Binder IPC facilities, iOS offers very few IPC (Inter Process Communication) options, minimizing the potential attack surface.
 
-Uniform hardware and tight hardware/software integration create another security advantage. Every iOS device offers security features, such as secure boot, hardware-backed keychain, and file system encryption (referred as data protection in iOS). iOS updates are usually quickly rolled out to a large percentage of users, decreasing the need to support older, unprotected iOS versions.
+Uniform hardware and tight hardware/software integration create another security advantage. Every iOS device offers security features, such as secure boot, hardware-backed Keychain, and file system encryption (referred as data protection in iOS). iOS updates are usually quickly rolled out to a large percentage of users, decreasing the need to support older, unprotected iOS versions.
 
-In spite of the numerous strengths of iOS, iOS app developers still need to worry about security. Data protection, Keychain, Touch ID/Face ID authentication, and network security still leave a large margin for errors. In the following chapters, we describe iOS security architecture, explain a basic security testing methodology, and provide reverse engineering how-tos.
+In spite of the numerous strengths of iOS, iOS app developers still need to worry about security. Data protection, Keychain, Touch ID/Face ID authentication, and network security still leave a large margin for errors. In the following chapters, we describe iOS security architecture, explain a basic security testing methodology, and provide reverse engineering How Tos.

```

Comment:
this looks a little strange to me...

Reply:
Well, the original looked weird so I looked it up, and this random guy on SE said you can use either:
https://english.stackexchange.com/questions/154563/what-is-the-plural-of-how-to

Then again, Wikipedia seems to have a different opinion (https://en.wikipedia.org/wiki/How-to). How about I change it to "guides"? ;)

Reply:
please do :D

Reply:
Scratch that: let's say how-tos?

Reply:
I would also say let's stick with how-tos 

https://www.wordhippo.com/what-is/the-plural-of/how_to.html

Otherwise "guides" is also fine, in case we cannot agree ;-)

Reply:
Fine, you guys win ;)